### PR TITLE
override new phpcs rule

### PR DESCRIPTION
**Pull request by mathroc**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ matrix:
   fast_finish: true
   include:
     - php: 5.3
+      dist: precise
```
Comment (mathroc) [inline .travis.yml]: because https://docs.travis-ci.com/user/reference/trusty#PHP-images

**Comment by erayd** [comment]
Won't forcing this to post, rather than simply disabling it, potentially introduce logic errors in the future? Changing the order of an increment / decrement operation in any situation that assigns the result somewhere is asking for trouble, as they are not equivalent.

Can we turn this rule off completely so that it leaves increment / decrement alone? I don't trust php-cs-fixer to recognize assignment situations, and it's introduced problems with this kind of assumption in the past.

**Comment by erayd** [comment]
Note the comments above regarding assignment also include using the result as input to another logic operation, regardless of whether a variable is explicitly set.

File: .php_cs.dist [hunk 2]
```diff
@@ -24,6 +24,7 @@ $config
         'pre_increment' => false,
         'trailing_comma_in_multiline_array' => false,
         'simplified_null_return' => false,
+        'increment_style' => false,
```
Comment (erayd) [inline .php_cs.dist]: Thanks :-)

**Comment by mathroc** [comment]
that’s true, it disabled it now, unfortunately it seems there is issue when running `composer install` with some version of php. might be related to #463 ?

**Comment by erayd** [comment]
> ...unfortunately it seems there is issue when running composer install with some version of php. might be related to #463 ?

Hooray, more dependency issues! Will take a look, hang on.

File: .travis.yml (hunk 1 above)
Comment (erayd) [reply .travis.yml]: This is already fixed in 6.0.0-dev.

**Comment by mathroc** [comment]
my bad, I even saw the comment https://github.com/justinrainbow/json-schema/pull/390#issuecomment-339419771 a few days ago and didn’t remember…, so that’s much better now

I don’t have much time to help more so I just wanted to do this to get some work of your plate (even if it’s not much), thx for the all work ;)

**Comment by erayd** [comment]
All good - I appreciate your taking care of this :-).

**Pull request merged**